src/EditorFeatures/TestUtilities/Workspaces/TestHostProject.cs

```diff
@@ -344,28 +344,24 @@ internal HostLanguageServices LanguageServiceProvider
         public ProjectInfo ToProjectInfo()
         {
             return ProjectInfo.Create(
-                this.Id,
-                this.Version,
-                this.Name,
-                this.AssemblyName,
-                this.Language,
-                this.FilePath,
-                this.OutputFilePath,
-                outputRefFilePath: null,
-                defaultNamespace: null,
-                this.CompilationOptions,
-                this.ParseOptions,
-                this.Documents.Select(d => d.ToDocumentInfo()),
-                this.ProjectReferences,
-                this.MetadataReferences,
-                this.AnalyzerReferences,
-                this.AdditionalDocuments.Select(d => d.ToDocumentInfo()),
-                this.AnalyzerConfigDocuments.Select(d => d.ToDocumentInfo()),
-                this.IsSubmission,
-                this.HostObjectType,
-                hasAllInformation: true,
-                runAnalyzers: true)
-                .WithDefaultNamespace(this.DefaultNamespace);
+                Id,
```

Comment:
fwiw, i far prefer the `this.` style as it is immediately unambiguous to me that this is referring to instance state.

Reply:
Our `.editorconfig` prefers the opposite.